### PR TITLE
API-8504 added source_type to register webhook

### PR DESCRIPTION
**Pull request by MaciejSzaflik**
## 🚀 Links

- [Feature branch](https://developers.labs.livechat.com/docs/feature/API-8504-source_type-filter-webhooks/)
- [Jira](https://livechatinc.atlassian.net/browse/API-8434)

## 📓 Description

Added info about new webhook filter (method register webhook and changelog). Only in version 3.4. 

### Content

- New content
- Proofreading/content fixes

### CODE REVIEW

File: content/management/configuration-api/v3.4/index.mdx [hunk 1]
```diff
@@ -2438,6 +2438,7 @@ Registers a webhook for the Client ID (application) provided in the request. Lat
 | `filters.chat_presence.user_ids.values`         | no       | `[]string` | Array of Users' ids; if any specified User is in the chat, the webhook will be triggered.                                                                                                      |
 | `filters.chat_presence.user_ids.exclude_values` | no       | `[]string` | Array of Users' ids; if any specified Users is in the chat, the webhook will not be triggered.                                                                                                 |
 | `filters.chat_presence.my_bots`                 | no       | `bool`     | If any bot owned by `owner_client_id` is in the chat, the webhook will be triggered.                                                                                                           |
+| `filters.source_type`                           | no       | `[]string` | Filters to check if a webhook should be triggered based on webhook's source. Possible values in array : `my_client`, `other_clients`, `system`.                                                |
```
Comment (litwinow) [inline content/management/configuration-api/v3.4/index.mdx:2441]: IMO this note should be similar to other filters'

Comment (MaciejSzaflik) [reply content/management/configuration-api/v3.4/index.mdx]: _Array of source types. Possible values: `my_client`, `other_clients`, `system`; if webhook source match, the webhook will be triggered._
something more like this?

Comment (litwinow) [reply content/management/configuration-api/v3.4/index.mdx]: How about
> Array of source types. Possible values: my_client, other_clients, system; if webhook's source client_id matches, the webhook will be triggered.

Comment (MaciejSzaflik) [reply content/management/configuration-api/v3.4/index.mdx]: Isn't it confusing that here we have source types, and here we are talking about client_id? Is it known that we're recognizing source of webhook by client_id?

Comment (litwinow) [reply content/management/configuration-api/v3.4/index.mdx]: maybe we'll need some additional explaination as `webhook source` is not defined anywhere and it's hard to deduct that it's connected to client_id of the request that triggered the webhook

Comment (oliwiapolec) [reply content/management/configuration-api/v3.4/index.mdx]: hey! I suggest:
> Array of source types. Possible values: `my_client` (client ID of your app), other_clients (client IDs other than your app's), system (no client ID); if the source which triggered the webhook matches the filter, the webhook will be sent.

This way we would guide the user on what is a webhook source without a separate disclaimer. I was a little concerned that my_client may not indicate that we refer to the client_id thus the explanations in brackets

**Review by oliwiapolec — APPROVED** [review]

File: content/management/changelog/index.mdx [hunk 2]
```diff
@@ -15,7 +15,9 @@ Since the migration is still ongoing, different parts will be successively moved
 
 ## [v3.4] - Developer preview
 
-No changes yet.
+### Webhooks
+
+- The `source_type` filter was added to all webhooks.
```
Comment (ale-kacperczyk) [inline content/management/changelog/index.mdx:20]: We could link the Register Webhook method under `source_type`

File: content/management/changelog/index.mdx [hunk 3]
```diff
@@ -15,7 +15,9 @@ Since the migration is still ongoing, different parts will be successively moved
 
 ## [v3.4] - Developer preview
 
-No changes yet.
+### Webhooks
+
+- The [`source_type`](/management/configuration-api/v3.3/#register-webhook) filter was added to all webhooks.
```
Comment (ale-kacperczyk) [inline content/management/changelog/index.mdx:20]:
```suggestion
- The [`source_type`](/management/configuration-api/v3.4/#register-webhook) filter was added to all webhooks.
```

**Review by ale-kacperczyk — APPROVED** [review]

File: content/management/changelog/index.mdx (hunk 3 above)
Comment (MaciejSzaflik) [reply content/management/changelog/index.mdx]: 🤦 my bad

**Review by litwinow — APPROVED** [review]

**Pull request merged**